app/src/main/res/layout/dashboard_list_item_category.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable name="category" type="String" />
+        <variable name="isSelected" type="Boolean" />
+        <variable name="callbacks" type="org.cru.godtools.ui.dashboard.tools.CategoryAdapterCallbacks"/>
+    </data>
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <androidx.cardview.widget.CardView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:minHeight="64dp"
+            android:layout_margin="4dp"
+            android:padding="@dimen/tools_card_corner_radius"
+            android:shadowColor="@color/tintable"
```

Comment:
this is the wrong color constant to use here, you aren't planning on tinting the shadow are you?